strings/names/first.txt

```diff
@@ -957,7 +954,6 @@ Mariabella
 Mariah
 Mariana
 Marilene
-Mario
```

Comment:
:+1: Iffy, but I agree its way too famous and mario has no commonly known last name